### DISCUSSION TITLE
Which configuration style do you use? XML, Java or a mix of both?

### DISCUSSION THREAD

**Discussion by fmbenhassine** [spring-projects/spring-batch#4677] [Polls]

**Comment by Sax388** [comment]
The only XML config I currently have is [Logback](https://logback.qos.ch/). I can't even read XML anymore 😆, spring config is also not XML, but YAML.

**Reply by fmbenhassine** [reply]
Got it, thank you for answering!

**Reply by deveshgarg150** [reply]
I also feel that no one likes to work with XML anymore (unless they are an AI agent/robot), given that they are so much prone to errors and usually in fast-paced work environments, you don't wanna spend so much time into XML hell itself!!
